x-pack/plugins/maps/common/descriptor_types/map_descriptor.ts

```diff
@@ -41,6 +42,7 @@ export type Goto = {
 export type TooltipFeature = {
   id: number;
   layerId: string;
+  mbProperties?: GeoJsonProperties;
```

Comment:
Can mbProperties be undefined? Where its used in TooltipPopover, `const properties = targetFeature ? targetFeature.properties : mbProperties;`, there is no check for undefined and where its populated, it should always be provided.

Reply:
I actually don't know.

"tags" (aka properties) are optional in the .mvt-spec (https://github.com/mapbox/vector-tile-spec/blob/master/2.0/README.md#44-feature-attributes), but I don't know how these are being normalized by mapbox-gl; as `null` (which is possible in GeoJson, https://tools.ietf.org/html/rfc7946#section-3.2), as `undefined`, or as empty object literal.

I'd suggest to be defensive and handle all three cases. I updated the typing.

Reply:
discussed. instead, will normalize properties up-front iso letting it bleed through type.